common/src/main/java/com/epam/solid/nie/ships/AbstractShipPositions.java

```diff
@@ -18,4 +19,17 @@
     void setPositions(List<Point2D> positions) {
         this.positions = positions;
     }
+
+    public BattleShipType getBattleShipType() {
+        return battleShipType;
+    }
+
+    public List<Point2D> getShipProperties() {
```

Comment:
why method called get if it is not a getter, also why shipProperties if its return position?